src/App.js

```diff
@@ -1,25 +1,42 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from "react";
+//components
+import Search from "./components/Search";
+import DefaultCity from "./components/DefaultCity";
+import Weather from "./components/Weather";
+import WeatherData from "./components/WeatherData";
+//style
+import GlobalStyle from "./gloablStyle";
+//redux
+import { useDispatch } from "react-redux";
+import {
+  makeDefaultCity,
+  loadDefaultCityForecast,
+  loadDefaultCityCurrent,
+  loadIsLoading,
+} from "./actions/makeDefaultAction";
+
+const App = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (localStorage.getItem("city")) {
+      dispatch(loadIsLoading());
+      dispatch(makeDefaultCity(localStorage.getItem("city")));
+      dispatch(loadDefaultCityForecast(localStorage.getItem("city")));
+      dispatch(loadDefaultCityCurrent(localStorage.getItem("city")));
+    }
+  }, [dispatch]);
 
-function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <GlobalStyle />
```

Comment:
Nice! We'll used styled components in the future - it's my go-to for styling.

This is a good place `<GlobalStyle />' assuming everything will be rendered inside of `App` or you may want to consider wrapping it around `App`